addons/udes_stock/tests/test_picking_batch.py

```diff
@@ -1705,3 +1705,213 @@ def test04_rereservation_for_same_batch(self):
 
         batch.reserve_pallet('UDES11111')
         batch.reserve_pallet('UDES11111')
+
+
+class TestMultipleOrders(common.BaseUDES):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.pack_4apples_info = [{'product': cls.apple,
+                                  'qty': 4}]
+        cls.pack_4bananas_info = [{'product': cls.banana,
+                                  'qty': 4}]
+
+        cls.picking_type_pick.u_reserve_pallet_per_picking = True
+
+    def setUp(self):
+        super().setUp()
+        self.create_quant(self.apple.id,
+                          self.test_location_01.id,
+                          300,
+                          package_id=self.create_package().id)
+        self.create_quant(self.banana.id,
+                          self.test_location_02.id,
+                          300,
+                          package_id=self.create_package().id)
+
+    def tearDown(self):
+        super().tearDown()
+        self.picking_type_pick.u_max_reservable_pallets = 10
```

Comment:
Is this really needed? any change to the picking type should be rolled back when test ends.